### ISSUE TITLE
create ProMa Checklist from Purchase Order in bulk

### ISSUE THREAD

**Issue by wojosc** [phamos-eu/ProMa#102]
We need an option to be able to bulk create ProMa Checklists from Purchase Orders

This is mostly used, when Purchase Orders are updated from import file through Portatour.

**Comment by roquegv** [comment]
This could be done adding a button in Purchase Order list view that will be shown when selecting one or more orders from the list. This button will create the ProMa Checklists in bulk, for each selected purchase order.

**Comment by roquegv** [comment]
this is solved by https://github.com/phamos-eu/ProMa/pull/94

![Kazam_screencast_00027](https://user-images.githubusercontent.com/6966715/234071640-0426505a-10a8-4352-8c6c-3b1a8ef1e875.gif)

NOTE: I had a problem with making changes to the Purchase Order's listview settings. So, I implemented a hacky way to simulate `frappe.lisview_settings["Purchase Order"]["onload"]` functionality, because it does not work on custom app.

**Comment by wojosc** [comment]
here is an example of where a similar solution is applied in the erpnext standard:
![grafik](https://user-images.githubusercontent.com/22279621/234265208-fbbce34e-cfab-4488-a50b-9f9cae2cc665.png)

Just for reference!

**Comment by roquegv** [comment]
> here is an example of where a similar solution is applied in the erpnext standard:

Yes, that was my first approach. The problem was that if I want to override the listview setting to add another menu button via custom app, then it does not work. It only works if I edit the erpnext's sales_order_list.js file, but we don't want that, so I needed to figure out a workaround.
